DQMServices/Core/interface/ConcurrentMonitorElement.h

```diff
@@ -69,6 +69,21 @@ class ConcurrentMonitorElement
     me_->ShiftFillLast(y, ye, xscale);
   }
 
+  // expose as a const method to mean that it is concurrent-safe
+  template <typename T>
+  void fillNoFlow(T val) const {
```

Comment:
@makortel can you elaborate on what is the goal ?

Reply:
To fill under/overflows to first/last bins (instead of under/overflow bins; as I tried to write in the PR descriptions). The function is not necessarily the best way to achieve the goal, but follows what MTV has "traditionally" done (ok, this sounds more an excuse than rationale; as a possibly better alternative I could think of somehow marking a MonitorElement such that e.g. when writing to disk the under/overflow bin contents get moved to first/last bins).